components/resources/commands.grdp

```diff
@@ -11,7 +11,7 @@
     ChromeOS) prefer to not add the command.
   -->
   <message name="IDS_IDC_COMMANDER">
-    Quick commands
+    Quick Commands
```

Comment:
This one is a bit trickier: we do want to capitalize it on MacOS, but not on other platforms. I think we may want to move it down in the file into `<if expr="use_titlecase">` section with capitalization and duplicate in `<if expr="not use_titlecase">` section without. cc: @fallaciousreasoning

Reply:
Yup, I think thats a good call :smile:

Reply:
Hello @mkarolin, @fallaciousreasoning,
I've added the conditional cases check. Please have a look.